pnparser/patch_notes.py

```diff
@@ -0,0 +1,625 @@
+from .exceptions import ParserError, ParserHttpError, ParserTimeoutError
+from .helpers import Helper, Filters
+from .dragon import Dragon
+from .templates import *
+
+from dateutil import parser as DatetimeParser
+from datetime import datetime as DateTime
+
+from bs4 import BeautifulSoup, Tag
+import requests as HttpClient
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from river_mwclient.esports_client import EsportsClient
+    from typing import Iterator
+
+# TODO: create statistics tab
+
+RIOT_ADDRESS: str = "https://na.leagueoflegends.com/en-us/news/game-updates/patch-{}-notes"
+SUMMARY: str = "Auto-parse League of Legends patch notes."
+WIKI_PAGE: str = "User:Bruno_Blanes/Patch_{}"
+
+# TODO: ask for help with this shit
+# known champion ability base attributes
+def ability_base_attributes() -> 'list[str]':
+    return ["Cooldown",
+            "First Hit Bonus Damage",
+            "Damage",
+            "Cost",
+            "Move Speed",
+            "Second Hit Healing Vs. Minions",
+            "Toggle Abilities Bugfix",
+            "Hecarim Bugfix"]
+
+
+class PatchNotes:
+    def __init__(self, site: 'EsportsClient') -> None:
+        self.context: str = ""
+        self.page_url: str = ""
+        self.patch_version: str = ""
+        self.site: 'EsportsClient' = site
+        self.patch_url: 'str | None' = None
+        self.sections: 'list[Section]' = []
+        self.published_date = DateTime.now()
+        self.designers: 'list[Designer]' = []
+        
+    def __print(self) -> None:
```

Comment:
private methods should be denoted with just 1 leading `_` since 2 `__` denotes "special" methods (like `__init__` etc`)

Reply:
Fixed.